game/views.py

```
@@ -620,7 +627,17 @@ def ai_move(request):
             response['Retry-After'] = str(window)
             return response
 
-    game_data = request.session.get('game')
+    client_version = None
+    try:
+        data = json.loads(request.body)
+        client_version = data.get('version')
+    except Exception:
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)